dvc/repo/plots/template.py

```diff
@@ -300,27 +302,95 @@ class NormalizedConfusionTemplate(Template):
     }
 
 
-class DefaultScatterTemplate(Template):
```

Comment:
`Default` in the name does not make sense, its not default in any way other than, all plots prepared by us are "default" for dvc.